src/text.rs

```diff
@@ -31,6 +31,12 @@ pub fn escape(data: &[u8], buf: &mut String) {
     buf.push('"');
 }
 
+pub fn hex_escape(data: &[u8], buf: &mut String) {
+    buf.push_str("hex(");
```

Comment:
why not 0x directly?

Reply:
`0x` may lead to some confusion due to Endianness. For example, it is not clear that `0xFFC0` is `[0xFF, 0xC0]` or `[0xC0, 0xFF]`, while `FFC0` mostly does not.

Reply:
Em, I think we don't need to care Endianness here. here I don't like the output contains a "hex()", too verbose.

how about the Go output? @nolouch

Reply:
Mostly formats like 0xFFC0 will give you a misconception that it is a number, but in fact it is not. If you treat it as a number in most programming languages then you will get a wrong endianness (since x86 is little endian and 0xFFC0 is C0, FF actually).

Reply:
here I just don't like "hex()".

Reply:
I don't think it's compatible with PD. The key print by PD doesn't have `hex()` wrapper.

>  If you treat it as a number in most programming languages then you will get a wrong endianess

It's not the same to write number in hex format and convert from bytes array. When writing number in hex format, Endianess don't have to be taken into account. For example, when you write 0xFFC0 in any code, you will get the same number 65472.

Reply:
any update? @breeswish

Reply:
@BusyJay So what's your idea about the format?